templates/overview/cards/irus-card.jsx

```diff
@@ -24,7 +25,13 @@ const Content = ({ data, ...htmlProps }) => (
 
 const IrusCard = ({ statistics }) => (
   <OverviewCard>
-    <Card.Title tag="h2">IRUS</Card.Title>
+    <Card.Title tag="h2">Donwload statistics</Card.Title>
```

Comment:
typo 🐛

Reply:
Yeah, I often do this.
```suggestion
    <Card.Title tag="h2">Download statistics</Card.Title>
```